src/components/SearchBox/SearchResults.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react'
 import styled from 'styled-components/macro'
-import { FixedSizeList } from 'react-window'
-import AutoSizer from 'react-virtualized-auto-sizer'
+import { Virtuoso } from 'react-virtuoso'
```

Comment:
If this lib candle the list items changing height when expanding/collapsing the graph, we could use it in the node list as well